llama_stack/distribution/build_container.sh

```diff
@@ -336,7 +336,7 @@ $CONTAINER_BINARY build \
   "$BUILD_CONTEXT_DIR"
 
 # clean up tmp/configs
-rm -f "$BUILD_CONTEXT_DIR/run.yaml"
+rm -rf "$BUILD_CONTEXT_DIR/run.yaml" "$TEMP_DIR"
```

Comment:
As a followup, or here if you feel like it, we really need a "trap" on ERR to cleanup these files ;)